xbmc/settings/windows/GUIControlSettings.cpp

```diff
@@ -543,7 +544,7 @@ bool CGUIControlButtonSetting::OnClick()
     }
     else if (controlFormat == "path")
       SetValid(GetPath((CSettingPath *)m_pSetting));
-    else if (controlFormat == "action")
+    else if (controlFormat == "action" || controlFormat == "infoaction")
```

Comment:
Will you make use of the `OnSettingAction()` callback for an `infoaction` setting or is it just meant to be a label/text?